### ISSUE TITLE
Filters

### ISSUE THREAD

**Issue by IgnazNepumuk** [azachar/slackline-festivals#1]
It would be nice, if there is a way to filter more than one country or a whole continent.

**Comment by azachar** [comment]
So let me rephrase it; you would like to see only events in Europe or in South America and so on. Is that right?  Actually I was thinking to provide a world map with all events so you can find your events near by you. Would it this solve your problem?

**Comment by IgnazNepumuk** [comment]
Yes , right. 

Maybe its a good Idea to set a radius where you can search within. (50,100, 500 km, or so)

**Comment by azachar** [comment]
Sounds good, but at this time, there are not that many events, perhaps when it starts to be an "hot" topic we can do this.

**Comment by IgnazNepumuk** [comment]
seems legit.

yout don´t want to upload your code here?

**Comment by azachar** [comment]
I am sorry, but this github repo is only for tracking feedback.

**Comment by IgnazNepumuk** [comment]
Is there a good reason for doing this ?

**Comment by azachar** [comment]
Yes, there are multiple reasons. Are you interested in developing it? If so, write me a private message.

**Comment by azachar** [comment]
The new release contains a special continent filtering:

![image](https://cloud.githubusercontent.com/assets/4659019/5485392/c8655958-869a-11e4-90fb-08ca3104b36a.png)

So I hope you will enjoy it.
